simplq/src/styles/joinPage.module.scss

```diff
@@ -13,3 +17,39 @@
     margin: 1rem;
   }
 }
+.header-title {
+  margin: auto;
+  .sub-header {
+    display: flex;
+    font-family: Pacifico;
+    font-size: 0.8rem;
+    font-weight: 300;
+    color: #3a3768;
+    align-items: center;
+  }
+}
+
+.header-bar {
+  @include background-color();
+  padding: 0.5rem;
+  // padding-top: 4rem;
+  
+  display: flex;
+  flex-wrap: wrap;
+  .simplq-logo{
+    margin-top: 35px;
+  }
```

Comment:
Can you replace the px with rem? We prefer using rem as the sizing unit everywhere.

Reply:
And also add some more padding to the left in order to match the design.

![Screenshot from 2020-10-06 13-25-48](https://user-images.githubusercontent.com/12583528/95174264-7a879100-07d7-11eb-8385-794c379e78cc.png)

Reply:
These are minor thing, but the heading is not correctly centred it seems.
![Screenshot from 2020-10-06 13-27-18](https://user-images.githubusercontent.com/12583528/95174506-cdf9df00-07d7-11eb-8835-3a32c59bd2e5.png)